src/components/logo/logo.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react"
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
+
+
+const Logo = ({                
+  path = "/",  
+  image,
```

Comment:
This prop isn't in the Logo.propTypes, is this correct?

Reply:
I think logo.js is not the Xmartlabs logo, so I tested it by deleting the document, and it didn't change anything, that's why I think it's the Gatsby logo. Do I delete the document or leave it anyway?